generation/WinSDK/enums.json

```diff
@@ -30286,11 +30046,27 @@
       }
     ]
   },
+  {
+    "name": "ISC_REQ_HIGH_FLAGS",
+    "namespace": "Windows.Win32.Security.Authentication.Identity",
+    "flags": true,
+    "type": "ulong",
+    "members": [
+      {
+        "name": "ISC_REQ_MESSAGES",
+        "value": "0x0000000100000000"
+      },
+      {
+        "name": "ISC_REQ_DEFERRED_CRED_VALIDATION",
+        "value": "0x0000000200000000"
+      }
+    ]
+  },
   {
     "name": "ISC_REQ_FLAGS",
     "flags": true,
     "autoPopulate": {
-      "filter": "ISC_REQ_",
+      "filter": "ISC_REQ_(?!MESSAGES|DEFERRED_CRED_VALIDATION)",
```

Comment:
Cool! I didn't know you could do that!

Reply:
I wasn't sure how to do it until I needed it. :)